src/newgrf_sound.cpp

```diff
@@ -22,6 +22,14 @@
 
 #include "safeguards.h"
 
+template <typename T>
+T* grow(std::vector<T>& vec, std::size_t size)
```

Comment:
third time i've seen this function now. Should be inlined for its one usage, or put somewhere generic and reused properly